tests/unit/test_source_suppress.py

```diff
@@ -1,169 +0,0 @@
-        self.assertEqual(sp_handler.suppress_comment(), 'some comment')
-
-    def test_multi_liner_all_2(self):
-        """There is suppress comment above the bug line."""
-        sp_handler = SourceSuppressHandler(self.__tmp_srcfile_1,
-                                           36,
-                                           "0",
-                                           "")
-        res = sp_handler.check_source_suppress()
-        self.assertTrue(res)
-        self.assertEqual(
-            sp_handler.suppressed_checkers(), {'my.checker_1', 'my.checker_2'})
-        self.assertEqual(
-            sp_handler.suppress_comment(), 'some really long comment')
-
-    def test_one_liner_some_checkers(self):
-        """There is suppress comment above the bug line."""
-        sp_handler = SourceSuppressHandler(self.__tmp_srcfile_1,
-                                           43,
-                                           "0",
-                                           "")
-        res = sp_handler.check_source_suppress()
-        self.assertTrue(res)
-        self.assertEqual(
-            sp_handler.suppressed_checkers(), {'my.Checker_1', 'my.Checker_2'})
-        self.assertEqual(
-            sp_handler.suppress_comment(), 'some really really long comment')
-
-    def test_multi_liner_some_checkers(self):
-        """There is suppress comment above the bug line."""
-        sp_handler = SourceSuppressHandler(self.__tmp_srcfile_1,
-                                           50,
-                                           "0",
-                                           "")
-        res = sp_handler.check_source_suppress()
-        self.assertFalse(res)
-        self.assertEqual(sp_handler.suppressed_checkers(), set())
-        self.assertIsNone(sp_handler.suppress_comment())
-
-    def test_comment_characters(self):
-        """Check for different special comment characters."""
-        sp_handler = SourceSuppressHandler(self.__tmp_srcfile_1,
-                                           57,
-                                           "0",
-                                           "")
-        res = sp_handler.check_source_suppress()
-        self.assertTrue(res)
-        self.assertEqual(
-            sp_handler.suppressed_checkers(), {'my.checker_1', 'my.checker_2'})
-        self.assertEqual(sp_handler.suppress_comment(), "i/';0 (*&^%$#@!)")
-
-    def test_fancy_comment_characters(self):
-        """Check fancy comment."""
-        sp_handler = SourceSuppressHandler(self.__tmp_srcfile_1,
-                                           64,
-                                           "0",
-                                           "")
-        res = sp_handler.check_source_suppress()
-        self.assertTrue(res)
-        self.assertEqual(sp_handler.suppressed_checkers(), {'my_checker_1'})
-        self.assertEqual(
-            sp_handler.suppress_comment(),
-            "áúőóüöáé ▬▬▬▬▬▬▬▬▬▬ஜ۩۞۩ஜ▬▬▬▬▬▬▬▬▬▬")
```

Comment:
This test case about handling special characters properly in the comments went missing.

Reply:
Only the file name was changed: 
https://github.com/csordasmarton/codechecker/blob/44bf1eeecc36da68d986d35e44e9166c2dc36051/tests/unit/test_source_code_comment.py#L161

Reply:
Yeah, my bad. GitHub didn't open the brand new file automatically because "It's too long".